src/main/java/com/pretz/dyntreeserver/service/Seed.java

```diff
@@ -0,0 +1,5 @@
+package com.pretz.dyntreeserver.service;
+
+public class Seed {
+    protected static final String SEED = "asfsdgssgs";
```

Comment:
I suggest moving this value to the (secret?) properties file and then inject it here with [@Value](http://www.baeldung.com/spring-value-annotation).

Or maybe even in such case delete this file and just directly inject it where needed.

In case if it's needed in many places maybe it'd be a good idea to create a 'proxy' configuration Component containing all properties values, e.g.:
```
public class Config {
    @Value('foo')
    String foo;
    [...]
}
```

and then inject it to `Services` etc. (but I've no idea if that's the "Spring way", maybe just putting `@Value` wherever needed is fine)

Reply:
Moved to properties and injected.